### PR TITLE
doc(hertz): add response writer ext doc

### PR DESCRIPTION
**Pull request by a631807682**
#### What this PR does / why we need it (en: English/zh: Chinese):

en: add en docs for response writer ext
zh: 添加response writer扩展的英文文档

#### Which issue(s) this PR fixes:

related to https://github.com/cloudwego/hertz/issues/656

### CODE REVIEW

File: content/en/docs/hertz/tutorials/framework-exten/response_writer.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: "Response Writer Extension"
+linkTitle: "Response Writer Extension"
+date: 2023-03-10
+weight: 2
```
Comment (GuangmingLuo) [inline content/en/docs/hertz/tutorials/framework-exten/response_writer.md:5]: need to be 4

**Comment by GuangmingLuo** [comment]
Thanks for your contribution. Can you add the zh version of this doc in the same PR ? @a631807682

**Comment by a631807682** [comment]
> Thanks for your contribution. Can you add the zh version of this doc in the same PR ? @a631807682

I will add zh doc later

File: content/en/docs/hertz/tutorials/framework-exten/response_writer.md (hunk 1 above)
Comment (a631807682) [reply content/en/docs/hertz/tutorials/framework-exten/response_writer.md]: done

File: content/zh/docs/hertz/tutorials/framework-exten/response_writer.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+weight: 4
+description: >
+
+---
+
+Hertz 提供了 response 的 writer 扩展， 用户可以根据自己的需要实现相应的接口去劫持 response 的 writer。
+
+## 接口定义
+
+接口定义在 `pkg/network/write`.
```
Comment (welkeyever) [inline content/zh/docs/hertz/tutorials/framework-exten/response_writer.md:14]: write -> writer

**Review by welkeyever — APPROVED** [review]
LGTM, thx~

**Pull request merged**